birdhouse/docker-compose.yml

```diff
@@ -101,16 +101,21 @@ services:
     restart: always
 
   geoserver:
-    image: pavics/geoserver:2.9.3
+    image: ${GEOSERVER_IMAGE}
     container_name: geoserver
     ports:
       - "8087:8080"
     environment:
       # for custom entrypoint
       PAVICS_FQDN_PUBLIC: $PAVICS_FQDN_PUBLIC
+      STABLE_EXTENSIONS: ${GEOSERVER_STABLE_EXTENSIONS}
```

Comment:
Maybe use a constant style of passing env variables. Eg: all with `${}` or not.

Reply:
Done in 0c74cf3c12d1edf172563fb95e0e03e248502af2, thanks.